### PR TITLE
Fix missing dependency on enum34 package

### PR DESCRIPTION
**Pull request by tescande**

### CODE REVIEW

File: SPECS/python-linstor.spec [hunk 1]
```diff
@@ -10,6 +10,8 @@ BuildRequires: python-devel
 BuildRequires: python-setuptools
 BuildRequires: python-enum34
 
+Requires:  python-enum34
```
Comment (Wescoeur) [inline SPECS/python-linstor.spec]: Two spaces? :p

Comment (tescande) [reply SPECS/python-linstor.spec]: Damn !

File: SPECS/python-linstor.spec [hunk 2]
```diff
@@ -10,6 +10,8 @@ BuildRequires: python-devel
 BuildRequires: python-setuptools
 BuildRequires: python-enum34
```
Comment (stormi) [inline SPECS/python-linstor.spec]: Is the BuildRequires still needed?

Comment (tescande) [reply SPECS/python-linstor.spec]: No, and it's not even a build requirement. Removing it...

**Review by stormi — APPROVED** [review]

**Review by Wescoeur — APPROVED** [review]

**Pull request merged**